BlackBox/BlackBox/BridgeDFA.ino

```diff
@@ -0,0 +1,105 @@
+// Bridge states
+State AA = State(InStateAction);
+State AW = State(InStateAction);
+State WA = State(InStateAction);
+State AB = State(InStateAction);
+State WW = State(InStateAction);
+State BA = State(InStateAction);
+State WB = State(InStateAction);
+State BW = State(InStateAction);
+State BB = State(InStateAction);
+
+void RunBridgeDFA(char input[]) {
+  FSM DFA = FSM(AA);
+  bool stringComplete = false;
+  DFA.update();
+
+  for (int i = 0; i < strlen(input); i += 2) {
+    if (input[i] == 'A') {
+      if (input[i + 1] == 'w') {
+        if (DFA.isInState(AA)) {
+          DFA.transitionTo(AW);
+        } else if (DFA.isInState(WA)) {
+          DFA.transitionTo(WW);
+        } else if (DFA.isInState(BA)) {
+          DFA.transitionTo(BW);
+        }
+      } else if (input[i + 1] == 'e') {
+        if (DFA.isInState(AA)) {
+          DFA.transitionTo(WA);
+        } else if (DFA.isInState(AW)) {
+          DFA.transitionTo(WW);
+        } else if (DFA.isInState(AB)) {
+          DFA.transitionTo(WB);
+        }
+      } else {
+        Serial.println("Failed because it enters the trash state");
+        break;
+      }
+    } else if (input[i] == 'E') {
+      if (input[i + 1] == 'w') {
+        if (DFA.isInState(AW)) {
+          DFA.transitionTo(AB);
+        } else if (DFA.isInState(WW)) {
+          DFA.transitionTo(WB);
+        } else if (DFA.isInState(BW)) {
+          DFA.transitionTo(BB);
+        }
+      } else if (input[i + 1] == 'e') {
+        if (DFA.isInState(WA)) {
+          DFA.transitionTo(BA);
+        } else if (DFA.isInState(WW)) {
+          DFA.transitionTo(BW);
+        } else if (DFA.isInState(WB)) {
+          DFA.transitionTo(BB);
+        }
+      } else {
+        Serial.println("Failed because it enters the trash state");
+        break;
+      }
+    } else if (input[i] == 'L') {
+      if (input[i + 1] == 'w') {
+        if (DFA.isInState(WB)) {
+          DFA.transitionTo(WA);
+        } else if (DFA.isInState(AB)) {
+          DFA.transitionTo(AA);
+        }
+      } else if (input[i + 1] == 'e') {
+        if (DFA.isInState(BW)) {
+          DFA.transitionTo(AW);
+        } else if (DFA.isInState(BA)) {
+          DFA.transitionTo(AA);
+        }
+      } else {
+        Serial.println("Failed because it enters the trash state");
+        break;
+      }
+    }
+
```

Comment:
I vores GA har jeg sat input symboler til char for at optimere, så her skal det også ændres til single letter input symbols ligesom i de to andre DFAs

Reply:
De nye input kan feks være A,B,C for train East Arrive, Enter, Leave og X,Y,Z for train west etc

Reply:
Det var det jeg foreslog til at starte med
![image](https://user-images.githubusercontent.com/72670627/231396051-73e08882-50ae-45e4-bfb7-f3a13585d2f5.png)

Ændrer det nu 👍